### ISSUE TITLE
[Feature Request] Active Effects support

### ISSUE THREAD

**Issue by Marcloure** [StasTserk/foundry-burningwheel#508]
It would be very nice to be able to use Foundry's Active Effects in Burning Wheel. This would make temporary modifiers such as spells and poison much easier to handle.

**Comment by StasTserk** [comment]
Hey, appreciate the suggestion. It's something I've been passively thinking about since active effects were announced as a concept in foundry.

Given the current [state of the system's development lifecycle](https://github.com/StasTserk/foundry-burningwheel?tab=readme-ov-file#state-of-development), so, while I agree it adding active effect support would open up a lot of possibilities (honestly, the way traits are implemented, they are just a bootleg version of what active effects became), the surest way to get them into the system is to write the code and open up a PR.

That said, I do have another week of vacation left, so I might have a look at how doable this change might prove to be.

TLDR; this is probably the best time to have put in this feature request, but if I can't bang something out in a week it might be a whole before anything can be done about this.
